package.json

```diff
@@ -60,10 +64,18 @@
     "semver": "^5.1.0",
     "tslint": "^5.0.0",
     "tslint-config-standard": "^6.0.1",
-    "typescript": "^2.4.1",
-    "typings": "^2.0.0"
+    "typescript": "^2.4.1"
   },
   "dependencies": {
+    "@types/arrify": "^1.0.1",
+    "@types/chalk": "^0.4.31",
+    "@types/diff": "^3.2.1",
+    "@types/minimist": "^1.2.0",
+    "@types/mkdirp": "^0.5.0",
+    "@types/node": "^6.0.88",
+    "@types/source-map-support": "^0.4.0",
+    "@types/yn": "types/npm-yn",
```

Comment:
Can we put the commit hash for the GitHub installed dependencies? Otherwise we might get a major version that's incompatible later and we won't want that to break everything 😄

Reply:
Sure!